tensorboard/webapp/widgets/data_table/header_cell_component.ng.html

```diff
@@ -60,15 +46,12 @@
     ></mat-icon>
   </div>
   <div
-    *ngIf="header.removable || header.sortable"
-    class="context-menu-container"
+    *ngIf="(header.removable || header.sortable) && !disableContextMenu"
+    class="context-menu-container show-on-hover"
   >
-    <button
-      mat-icon-button
-      class="context-menu-trigger"
+    <mat-icon
```

Comment:
+1 for accessibility, that is why it was a button originally.
I made it just an icon so that it would be consistent with the sort icon, should I also make the sort icon a button too? Also I thought that people didn't like the ripple which comes with mat-button